server/app/controllers/admin/ProgramMigration.java

```diff
@@ -0,0 +1,31 @@
+package controllers.admin;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import services.program.ProgramDefinition;
+
+/**
+ * A Jackson serializable class used to migrate programs between instances. {@link
```

Comment:
**suggestion**: Can we say more about why this class is necessary? Is it a wrapper class to make serialization easier? If so can we name it like `ProgramMigrationWrapper`?